bundles/io/org.eclipse.smarthome.io.transport.dbus/about.html

```diff
@@ -1,48 +0,0 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
-    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml">
-<head>
-<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1"/>
-<title>About</title>
-</head>
-<body lang="EN-US">
-<h2>About This Content</h2>
- 
-<p>June 5, 2006</p>	
-<h3>License</h3>
-
-<p>The Eclipse Foundation makes available all content in this plug-in (&quot;Content&quot;).  Unless otherwise 
-indicated below, the Content is provided to you under the terms and conditions of the
-Eclipse Public License Version 1.0 (&quot;EPL&quot;).  A copy of the EPL is available 
-at <a href="http://www.eclipse.org/legal/epl-v10.html">http://www.eclipse.org/legal/epl-v10.html</a>.
-For purposes of the EPL, &quot;Program&quot; will mean the Content.</p>
-
-<p>If you did not receive this Content directly from the Eclipse Foundation, the Content is 
-being redistributed by another party (&quot;Redistributor&quot;) and different terms and conditions may
-apply to your use of any object code in the Content.  Check the Redistributor's license that was 
-provided with the Content.  If no such license exists, contact the Redistributor.  Unless otherwise
-indicated below, the terms and conditions of the EPL still apply to any source code in the Content
-and such source code may be obtained at <a href="http://www.eclipse.org/">http://www.eclipse.org</a>.</p>
-
-        <h3>Third Party Content</h3>
-        <p>The Content includes items that have been sourced from third parties as set out below. If you 
-        did not receive this Content directly from the Eclipse Foundation, the following is provided 
-        for informational purposes only, and you should look to the Redistributor's license for 
-        terms and conditions of use.</p>
-
-        <p>
-        <strong>libdbus-java-2.7.jar</strong> <br/><br/>
-        This bundle includes a library for accessing the linux DBUS from Java. The sources can be found in the
-        <a href="https://cgit.freedesktop.org/dbus/dbus-java/">Freedesktop Git repository</a>.
-        The copyright holders have granted the Eclipse Foundation the explicit permission to distribute the
```

Comment:
We had to add this specific sentence here at the time as the official project license is NOT the MIT Expat. You should therefore keep this in the NOTICE file as well.